samples/excel/excel-library/overview/Pages/ExcelLibraryOverview.razor

```diff
@@ -0,0 +1,255 @@
+@using Microsoft.AspNetCore.Components
+@using Microsoft.AspNetCore.Components.Rendering
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.RenderTree
+@using Microsoft.AspNetCore.Components.Web
+@using System.Text.RegularExpressions;
+@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.WebAssembly.Http
+@using Microsoft.JSInterop
+@using Infragistics.Documents.Excel
+
+<div class="igContainer">
+
+    <div class="igOptions">
+        <button @onclick="CreateXlsx">Create XLSX</button>
+        <button @onclick="CreateXls">Create XLS</button>
+    </div>
+</div>
+
+@code {
+
+    [Inject]
+    public IJSRuntime Runtime { get; set; }
+    public bool canSave = false;
+    public Workbook wb;
+    public Worksheet ws;
+    public List<string> worksheetRegion = null;
+    public string selectedRegion = null;
+    private Random Rand = new Random();
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+
+        this.WorkbookCreate();
+    }
+
+    private void CreateXls()
+    {
+        wb.SetCurrentFormat(WorkbookFormat.Excel97To2003);
+        
+        this.SaveFile(this.wb, "ExcelWorkbook");
+    }
+
+    private void CreateXlsx()
+    {
+        wb.SetCurrentFormat(WorkbookFormat.Excel2007);
+        
+        this.SaveFile(this.wb, "ExcelWorkbook");
+    }
+
+    private Infragistics.Documents.Excel.Workbook CreateWorkbook(Infragistics.Documents.Excel.WorkbookFormat format)
+    {
+        Infragistics.Documents.Excel.Workbook.InProcessRuntime = this.Runtime as IJSInProcessRuntime;
+
+        var wb = new Infragistics.Documents.Excel.Workbook(format);
```

Comment:
remove "Infragistics.Documents.Excel." because this namespace is already imported in the `using` statement

Reply:
resolved, pending update